runelite-client/src/main/scripts/ChatBuilder.rs2asm

```diff
@@ -216,9 +218,14 @@ LABEL183:
       91: LABEL207
    jump                   LABEL426
 LABEL186:
+   iload                  9
+   load_string            ""
+   load_string            "addTimestamp"
+   runelite_callback
+   pop_int
    sload                  9
    load_string            ":"
-   string_append          2
+   string_append          3
```

Comment:
Especially so for things like this - since they become very easy to miss in diffs during updates